docs/quickstart.rst

```diff
@@ -144,3 +144,111 @@ Otherwise, models are represented as dicts.
 
     Values in ``request.swagger_data`` are only available if
     ``pyramid_swawgger.enable_request_validation`` is enabled.
+
+Accessing Swagger Operation
+---------------------------
+
+During the implementation of an endpoint you could eventually have need of accessing the Swagger Specs that defined that specific view.
+``pyramid_swagger`` will inject in the request object a new property (that will be evaluated only if accessed) called ``operation``.
+
+``request.operation`` will be set to ``None`` for Swagger 1.2 defined endpoints, while it will be an `Operation <https://github.com/Yelp/bravado-core/blob/v4.8.4/bravado_core/operation.py#L12>`_ object if the endpoint is defined by Swagger 2.0 specs.
+
+:mod:`pyramid_swagger` renderer
+-------------------------------
+
+Using :mod:`pyramid_swagger` you will get automatic conversions of the input JSON objects to easy to handle python objects.
+An example could be a swagger object property marked to use date property, the library will take care of converting the
+ISO 8601 date representation to an easy to handle python ``datetime.date`` object.
+
+While defining the :mod:`pyramid` view that will handle the endpoint you have to make sure that the chosen renderer will be able to
+properly render you response. In the case of an endpoint that returns *objects* that requires a special handling
+(like ``datetime.date`` object) the developer is *forced* to:
+
+* perform manual operations to convert the python object to an object that could be handled by the renderer
+* add an `adapter <https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/renderers.html#using-the-add-adapter-method-of-a-custom-json-renderer>`_ for instructing pyramid to handle your object
+* define a custom renderer that is able to deal with this serialization problem
+
+:mod:`pyramid_swagger` (starting from version X.X.X) provides:
+
+* a new renderer, called ``pyramid_swagger``
+* a new renderer factory of factories, called ``pyramid_swagger.renderer.PyramidSwaggerRendererFactory``
+
+Using this new solutions you can reduce your efforts on dealing with formatting and focus more on your business logic.
+
+-----------------------------------------
+How :mod:`pyramid_swagger` renderer works
+-----------------------------------------
+
+The new :mod:`pyramid_swagger` renderer is a wrapper around the default ``pyramid.renderers.JSON`` renderer.
+
+:mod:`pyramid_swagger` will receive, from your pyramid view, the object that has to be rendered, perform the marshaling operations and then call the default JSON renderer.
+
+.. note::
+    The usage of this renderer will allow you to better use the Custom Formats configurable with ``pyramid_swagger.user_formats``.
```

Comment:
How? I'd add an explanation as well if you mention it. Otherwise you could say that only the pyramid_swagger renderer provides full support for custom formats...?